lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -12,106 +12,164 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
+import { HEADER_RID } from "../../constants";
 import { BaseRequest, BaseResponse } from "../../framework";
-import { TypeNormalisedInput } from "./types";
+import { TokenType, TypeNormalisedInput } from "./types";
 
+const authorizationHeaderKey = "authorization";
 const accessTokenCookieKey = "sAccessToken";
+const accessTokenheaderKey = "st-access-token";
 const refreshTokenCookieKey = "sRefreshToken";
+const refreshTokenHeaderKey = "st-refresh-token";
 
 // there are two of them because one is used by the server to check if the user is logged in and the other is checked by the frontend to see if the user is logged in.
 const idRefreshTokenCookieKey = "sIdRefreshToken";
-const idRefreshTokenHeaderKey = "id-refresh-token";
+const idRefreshTokenHeaderKey = "st-id-refresh-token";
```

Comment:
We still want to keep the header, but remove the cookie. As the header is to be saved on the frontend and will be required by it to do sync when refreshing a session.

Reply:
EDIT: We can actually get rid of this cause on the frontend, we can save (in cookies) the last time we did refreshing, and use that for sync.